pkg/exec/exec.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017 CNI authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exec
+
+import (
+	"io"
+	osexec "os/exec"
+)
+
+type cmdWrapper osexec.Cmd
```

Comment:
Why do we need this wrapper?  (I.e. the entire file)

Reply:
It was requested that it be wrapped by dcbw, I imagine for testing purposes.